test/ExecutionFramework.h

```diff
@@ -217,25 +217,25 @@ class ExecutionFramework
 			bytes const& ret = call(_name + "(string)", u256(0x20), _arg.length(), _arg);
 			BOOST_REQUIRE(ret.size() == 0x20);
 			BOOST_CHECK(std::count(ret.begin(), ret.begin() + 12, 0) == 12);
-			return eth::abiOut<u160>(ret);
+			return u160(u256(h256(ret)));
```

Comment:
Can the typecast work with skipping `u256`?

Reply:
Not sure, but I would also prefer doing it this way, because then we can be sure that we get the lower-endian part.